README.md

```diff
@@ -1,2 +1,127 @@
 # devices
 Servo-specific APIs to access various devices
+
+----------
+
+###How to use
+
```

Comment:
Also we should add an `Implementation/Desing/For Devs/someting_meaningfull_topic` and a `Usage` section. Because the embedder only interested in the usage, and the contributor in the design.